lib/opencensus/trace/integrations/rack_middleware.rb

```diff
@@ -24,7 +24,7 @@ def call env
           OpenCensus::Context.clear
           trace = OpenCensus::Trace.start
           begin
-            OpenCensus::Trace.in_span "rack-request" do |span|
+            OpenCensus::Trace.in_span "rack-request" do
```

Comment:
Shouldn't we still keep the argument for clarity, since it is yielded? Or does rubocop not like it when you don't use a block param? Can we rename it to `_span`?

Reply:
Yep, we can do this.